storage/tianmu/core/column_type.h

```diff
@@ -142,9 +142,12 @@ struct ColumnType {
   bool GetAutoInc() const { return flag[static_cast<int>(enumCT::AUTO_INC)]; }
   void SetAutoInc(bool inc) { flag[static_cast<int>(enumCT::AUTO_INC)] = inc; }
   bool HasFilter() const { return flag[static_cast<int>(enumCT::BLOOM_FILTER)]; }
+  bool GetUnsigned() const { return m_unsigned; }
+  void SetUnsigned(bool unsigned_) { m_unsigned = unsigned_; }
 
  private:
   common::ColumnType type;
+  bool m_unsigned = false;
```

Comment:
according to our variables naming style, we do use prefix `m_` to indicates it's a class member var.  you can use `is_unsigned` or something else.

Reply:
The `m_unsigned ` is  class member var